vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -62,120 +89,142 @@ ent ecr deploy --repo="https://github.com/entando-samples/standard-demo-manage-u
 
-4. Select `Deployed` and `Install` for each bundle, where order of installation is important. The `standard-demo-content-bundle` will need to be installed last, as it relies on MFEs from the other bundles to set up each of the pages. 
-   ![Installed.png](./images/Installed.png)
+     1. Click on the bundle entry
+     2. In the pop-up box, click "Deploy"
+     3. Click on the "Install" button that appears in this same pop-up box once the bundle deploys
+     4. Exit out of the pop-up box
+     5. Repeat steps 1-4 for each bundle
 
-Each installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets. The `standard-demo-banking-bundle` and `standard-demo-customer-bundle` include microservices that require the initialization of containers and will take longer to install.
+     ![sd-local-hub2.png](./sd-images/sd-local-hub2.png) 
 
-In the unlikely event you encounter conflicts during an initial installation, you will be presented with an Installation Plan like the one shown below. Select `Update All` in the upper right after making your selections.
-   ![InstallPlan.png](./images/InstallPlan.png)
+     **Order of installation is important.** The `standard-demo-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each of the pages.
 
+     >Note: The command line deploys and installs the Standard Banking Demo bundles without thumbnails.
 
-5. Access the Standard Banking Demo via one of the following options:
+     >Note: Each installation can take several minutes while the application downloads Linux images for the microservices and installs related assets. The `standard-demo-banking-bundle` and `standard-demo-customer-bundle` include microservices that require container initializations and take longer to install.
 
-**Option 1** If you'd like to make the Standard Banking Demo your default home page, go to `App Builder → Pages → Settings`. In
-   the dropdown for Home Page, select `Home / Home SD` and click `Save`.
-   ![HomepageSelect.png](./images/HomepageSelect.png)
+     ::: tip
+     In the unlikely event of conflicts during an initial installation, you will be presented with an Installation Plan like the one shown below. Select `Update All` in the upper right after making your selections.
+     ![InstallPlan.png](./images/InstallPlan.png)
+     :::
 
-You can now navigate to your application's home page using the home icon in the upper right of the App Builder.
+## Access the Standard Banking Demo
 
+Access the Standard Banking Demo via one of the following options.
 
-**Option 2** Alternatively, you can view the Standard Banking Demo home page by going to `Pages → Management`, finding `Home SD` in the page tree, and clicking `View Published Page` from its actions.
+### Option 1
 
+Make the Standard Banking Demo your default home page. The home icon shortcut will then navigate to the Standard Banking Demo home page. 
```

Comment:
take out `shortcut`

thought maybe there was another shortcut. think the home icon is sufficient.